Exec/SprayTests/PeleC/SprayA_wbreakup/GNUmakefile

```diff
@@ -0,0 +1,38 @@
+# AMReX
```

Comment:
Part of me is a bit worried about all these Exec tests in PeleMP. I can easily see us changing things in LMeX or C and forgetting that these exist. Maybe we should think about moving these problems into the actual code bases?